cookiecutter.json

```diff
@@ -2,8 +2,9 @@
     "project_name": "Hello World"
     , "project_description": "Add a short project description here."
     , "main_module": "{{ cookiecutter.project_name|lower|replace(' ', '_')|replace('-', '_') }}"
-    , "github_username": "Fueled"
-    , "github_repository": "{{ cookiecutter.main_module|replace('_', '-') }}-web"
+    , "git_hosting_service": "github"
+    , "username": "Fueled"
+    , "repository": "{{ cookiecutter.main_module|replace('_', '-') }}-web"
```

Comment:
I would suggest using the following variable: 

- `git_host` (default: github.com)
- `git_reponame` 
- `git_username`  (default: Fueled)
- `git_remote`: default( git@{{cookiecutter.git_host}}:{{cookiecutter.git_username}}/{{cookiecutter.git_reponame}}.git )


Alternatively, you can user a script in pre_hook to determine the different parts by parsing the git remote url.

Reply:
Noted. I also got to know that we can use `git mv` to rename directories in git. So seems like this PR this can be saved after all.

Reply:
Apparently `git mv` is a big fat lie. What do you think of asking these during code generation and injecting `github_repository` as an extra context in pre script ?